### ISSUE TITLE
I420VideoFrameStorage could not be found

### ISSUE THREAD

**Issue by truongtin** [microsoft/MixedReality-WebRTC#676]
I420VideoFrameStorage could not be found
I've added MixedReality.WebRTC.UWP in my UWP project. But I420VideoFrameStorage could not be found.
![image](https://user-images.githubusercontent.com/26736598/108479577-7bbde400-72c8-11eb-8ea9-392b3a7296e2.png)
I don't know how to fix this issue.
Please help me 
Thank you

**Comment by djee-ms** [comment]
I don't know either; you skipped the Issue Template with all the basic information about your project and environment, so I have no idea what you are trying to do on which platform with which version of the code.

**Comment by truongtin** [comment]
I follow the tutorial for UWP at https://microsoft.github.io/MixedReality-WebRTC/manual/cs/helloworld-cs-mediatracks-uwp.html.
I have a stuck at Add Local Media Tracks. When added existed VideoBridge.cs and StreamSamplePool.cs files. In VideoBridge.cs file, I cannot call I420VideoFrameStorage class

**Comment by djee-ms** [comment]
The class `I420VideoFrameStorage` is defined here:
https://github.com/microsoft/MixedReality-WebRTC/blob/8666f367e61fd478be5433b01939695a33f81d5b/libs/Microsoft.MixedReality.WebRTC/VideoFrameQueue.cs#L41

**Comment by truongtin** [comment]
I saw this code, I'll add this class into StreamSamplePool.cs file
Thank you

**Comment by djee-ms** [comment]
No, this code is in the core C# library, there is no need to add it anywhere.

**Comment by truongtin** [comment]
But I cannot call this class in my project. How do I call this class in StreamSamplePool.cs ?

**Comment by truongtin** [comment]
@djee-ms  Do you know where I can get these files? 
![image](https://user-images.githubusercontent.com/26736598/108682987-993aba00-7523-11eb-8687-5fd326a3faa6.png)
Thank you
